custom/app-setup/kafka/litmus-kafka-deployer/litmus-kafka-deployer-pod.yml

```diff
@@ -6,6 +6,7 @@ metadata:
     app: litmus-kafka-deployer
 spec:
   serviceAccountName: litmus-kafka-sa
+  restartPolicy: Never
```

Comment:
Let's keep `Always` as default

Reply:
If  we put Always here, then it will  restart the pod after the completion of pod